.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.6
+  - 2.1.10
```

Comment:
Actually, we have key for ruby 2.0 - you just removed it :-) https://github.com/rebelidealist/stripe-ruby-mock/blob/master/.travis.yml#L16